core/src/main/kotlin/model/CompositeSourceSetID.kt

```diff
@@ -0,0 +1,30 @@
+package org.jetbrains.dokka.model
+
+import org.jetbrains.dokka.DokkaConfiguration
+import org.jetbrains.dokka.DokkaSourceSetID
+
+data class CompositeSourceSetID(
```

Comment:
Naming alternatives: 
- `MultiSourceSetID` 
- `SourceSetIDComposite`

Reply:
`CompositeSourceSetID` is the best IMO.